.travis.yml

```diff
@@ -25,7 +25,7 @@ before_script:
 - minikube update-context
 # Fix the kubectl context, as it's often stale.
 # - minikube update-context
-- curl -Lo helm.tar.gz https://kubernetes-helm.storage.googleapis.com/helm-${HELM_VERSION}.tar.gz
+- curl -Lo helm.tar.gz https://get.helm.sh/helm-${HELM_VERSION}.tar.gz
```

Comment:
Just curious- could you explain the purpose of this change?

Reply:
Sure, for some reason the helm releases are no longer available in the previous URL.